src/org/zaproxy/zap/extension/pscanrules/CookieHttpOnlyScanner.java

```diff
@@ -66,32 +70,56 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		if (cookies2 != null) {
 			iterator.addIterator(cookies2.iterator());
 		}
+		
+		List<String> ignoreList = new ArrayList<String>();
+		String ignoreConf = getModel().getOptionsParam().getConfig().
```

Comment:
Since you added a `setModel()` function, would it be better to call that in `init()` and then use `model` here?